python/TestHarness/tests/test_CustomEval.py

```diff
@@ -0,0 +1,10 @@
+
+from TestHarnessTestCase import TestHarnessTestCase
+import subprocess
+class TestHarnessTester(TestHarnessTestCase):
+  def testCustomEval(self):
```

Comment:
See other comment - expand this testing to hit more lines of the additional python.